apps/admin-gui/src/app/users/pages/user-detail-page/user-dashboard/user-dashboard.component.ts

```diff
@@ -120,6 +120,9 @@ export class UserDashboardComponent implements OnInit {
     return new Promise((resolve, reject) => {
       if (this.guiAuthResolver.isOnlySponsor()) {
         const vosSponsoring = this.storeService.getPerunPrincipal().roles['SPONSOR']['Vo'];
```

Comment:
This line is where the problem occurs, to be more precise, in the last part `roles['SPONSOR']['Vo']` the Vo field might NOT be present. You need to detect if it is defined, not only the length.